tensorflow_lite_support/python/task/audio/pybinds/_pywrap_audio_embedder.cc

```diff
@@ -64,7 +64,11 @@ PYBIND11_MODULE(_pywrap_audio_embedder, m) {
       .def("get_embedding_dimension", &AudioEmbedder::GetEmbeddingDimension)
       .def("get_number_of_output_layers",
            &AudioEmbedder::GetNumberOfOutputLayers)
-      .def("get_required_audio_format", &AudioEmbedder::GetRequiredAudioFormat)
+      .def("get_required_audio_format",
+           [](AudioEmbedder& self) -> AudioBuffer::AudioFormat {
+             auto audio_format = self.GetRequiredAudioFormat();
+             return core::get_value(audio_format);
```

Comment:
As this may throw an error, you should mention that in the Python docstring.

Reply:
@khanhlvg Updated the property's docstring in both the Audio APIs.